### ISSUE TITLE
http subs filter header ignored, this may be a compressed response. while reading response header from upstream, client

### ISSUE THREAD

**Issue by TommehM** [yaoweibin/ngx_http_substitutions_filter_module#10]
2014/05/09 13:28:50 [warn] 19808#0: *312 http subs filter header ignored, this may be a compressed response. while reading response header from upstream, client: xx.xx.xx.xx, server: , request: "GET / HTTP/1.1", upstream: "http://194.71.107.80:80/", host: "tpb.rtbt.me", referrer: "http://tpb.rtbt.me/top/101"

**Comment by stucki** [comment]
Yes there is a problem: If the Content-Encoding is set to "none" then the module does still assume that the response is compressed.
I understand that the header is non-standard (https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.11), however it seems to be needed in some special cases. I'm currently testing that, however and for the time being, I think that ignoring "none" wouldn't hurt...
See also https://github.com/getgrav/grav/issues/548

**Comment by nagarciah** [comment]
Hi. Thanks a lot for your work, it's very valuable. Since compressed responses are a vast majority, are there plans to enable substitution module to work on compressed responses?.

**Comment by guitarama** [comment]
i worked around this one by removing the Accept-Encoding header going to proxy then enabling gzip on the proxied response, e.g.
```
    gzip_proxied any;
    ...
        proxy_set_header Accept-Encoding '';
        subs_filter ... ...;
```
